pom.xml

```diff
@@ -120,14 +215,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>${surefire.verion}</version>
-            </plugin>
-            <plugin>
-                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
why are we removing failsafe?

Reply:
Failsafe is usually used to run integration tests (tests named after these patterns https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html#inclusions).

This declaration isn't needed as long as there are no integration tests.